src/main/java/tatracker/logic/commands/FindCommand.java

```diff
@@ -12,12 +16,14 @@
  */
 public class FindCommand extends Command {
```

Comment:
i think this file can be deleted ? unless we want to have a find (specific student) command ?

Reply:
Okie!